apps/appsets/components.yaml

```diff
@@ -188,6 +188,27 @@ spec:
                     - repoURL: '{{index .metadata.annotations "uc_repo_git_url"}}'
                       targetRevision: '{{index .metadata.annotations "uc_repo_ref"}}'
                       path: 'components/chrony'
+                - component: openstack-exporter
+                  componentNamespace: monitoring
+                  skipComponent: '{{has "openstack-exporter" ((default "[]" (index .metadata.annotations "uc_skip_components") | fromJson))}}'
+                  sources:
+                    - repoURL: '{{index .metadata.annotations "uc_repo_git_url"}}'
+                      targetRevision: '{{index .metadata.annotations "uc_repo_ref"}}'
+                      path: 'components/openstack-exporter'
+                      ref: understack
+                    - repoURL: '{{index .metadata.annotations "uc_deploy_git_url"}}'
+                      targetRevision: '{{index .metadata.annotations "uc_deploy_ref"}}'
+                      ref: secrets
+                      path: 'secrets/{{.name}}/openstack-exporter'
+                    - repoURL: registry.scs.community/openstack-exporter
+                      chart: prometheus-openstack-exporter
+                      targetRevision: 0.4.5
+                      helm:
+                        releaseName: prometheus-openstack-exporter
+                        valueFiles:
+                          - $understack/components/openstack-exporter/values.yaml
```

Comment:
If we don't need this file then just drop it.

Reply:
We'll need it in the future, but I've removed it for now.